tool.go

```diff
@@ -588,7 +592,7 @@ func (fs serveCommandFileSystem) Open(requestName string) (http.File, error) {
 
 				sourceMapFilter := &compiler.SourceMapFilter{Writer: buf}
 				m := &sourcemap.Map{File: base + ".js"}
-				sourceMapFilter.MappingCallback = gbuild.NewMappingCallback(m, fs.options.GOROOT, fs.options.GOPATH)
+				sourceMapFilter.MappingCallback = gbuild.NewMappingCallback(m, fs.options.GOROOT, fs.options.GOPATH, false)
```

Comment:
doy.  sorry, I didn't read very well last time.

I've pushed a change that adds `--localmap` support to other commands:  specifically `install`, `test`, `serve`, and `get`.  Being a gopherjs noob, I wasn't quite sure how this flag would apply to `gopherjs get`, but it mentioned removing a `.map` file when browsing the source so I included it in this change.  Please advise whether I should remove it.